db/model/room.js

```diff
@@ -33,6 +34,14 @@ module.exports = function room(seq, dataTypes) {
       },
       comment: 'Create a named room ',
     },
+    settings: {
+      type: dataTypes.ARRAY(dataTypes.JSONB),
+      allowNull: true,
+      validate: {
+        contains: u.validateSettingsArray,
+      },
+      comment: 'Key/Value pairs for user specific settings',
```

Comment:
in this case i just copied it from the roomType as they are initialised from those settings and are then only changed on a room by room basis so it made most sense for it to be an array.